src/js/types/index.js

```diff
@@ -89,7 +89,8 @@ export type SearchRecord = {
   program: string,
   pins: string[],
   spanArgs: SpanArgs,
-  space: string
+  spaceID: string,
+  spaceName: string
```

Comment:
As mentioned in a comment earlier (near the search reducer). I think it would be difficult to maintain if we store the space name here along with the ID since the name can change at anytime.